docs/networking/simpleimpl.md

```diff
@@ -9,16 +9,12 @@ Getting Started
 First you need to create your `SimpleChannel` object. We recommend that you do this in a separate class, possibly something like `ModidPacketHandler`. Create your `SimpleChannel` as a static field in this class, like so:
 
 ```java
-private static final String PROTOCOL_VERSION = "1";
-public static final SimpleChannel INSTANCE = NetworkRegistry.newSimpleChannel(
-  new ResourceLocation("mymodid", "main"),
-  () -> PROTOCOL_VERSION,
-  PROTOCOL_VERSION::equals,
-  PROTOCOL_VERSION::equals
-);
+private static final int PROTOCOL_VERSION = 1;
+public static final SimpleChannel INSTANCE = ChannelBuilder.named(new ResourceLocation("mymodid","main"))
+            .networkProtocolVersion(PROTOCOL_VERSION)//set the version of channel.Optional,default is 0.Version can not be lower than 0.
```

Comment:
Put the comment above the line and split it between multiple lines when necessary.